### PR TITLE
Return of react broadcast

### PR DESCRIPTION
**Pull request by HowlingEverett**

### CODE REVIEW

File: packages/translated-components/src/index.js [hunk 1]
```diff
@@ -41,18 +27,27 @@ const translated = ({
 }) => {
   const warmTranslations = preHeat(translations, format)
```
Comment (langsharpe) [inline packages/translated-components/src/index.js:28]: WTH is a warm translation? 😁 🔥

Comment (bradparker) [reply packages/translated-components/src/index.js:28]: #performance

Comment (HowlingEverett) [reply packages/translated-components/src/index.js:28]: Honestly? I can't be sure what Ben meant 😛 .

At a *guess*, I'd say he means pre-processing the translations to interpolate their parameters and do any money-localisation, etc. Purely based on what I see the code doing. Why that's 'pre-warming' is anyone's guess.

Comment (langsharpe) [reply packages/translated-components/src/index.js:28]: Is it to warm up the...
` import kebabCase from 'lodash/kebabCase'`

Comment (HowlingEverett) [reply packages/translated-components/src/index.js:28]: Well I mean, nobody likes cold kebab meat.

We should just rename this library to HSP.

File: packages/ui-icons/package.json [hunk 2]
```diff
@@ -23,6 +23,7 @@
     "prop-types": "^15.5.10",
     "raw-loader": "^0.5.1",
     "react-svg-inline": "^2.0.0",
+    "standard": "^11.0.0",
```
Comment (bradparker) [inline packages/ui-icons/package.json]: How does this work as both a peer dep and a direct dep?

Comment (HowlingEverett) [reply packages/ui-icons/package.json]: It shouldn't be a direct dep - just a devDep. I'll update it. I'm bad at typing lerna commands.

**Review by bradparker — APPROVED** [review]

**Pull request merged**